### PR TITLE
gn4 - upgrade ES version

### PR DESCRIPTION
**Pull request by pmauduit**
See https://github.com/georchestra/georchestra/issues/3935

### CODE REVIEW

**Review by jahow — APPROVED** [review]
Looking good!

File: docker-compose.yml [hunk 1]
```diff
@@ -271,7 +271,7 @@ services:
       - ./config:/etc/georchestra
 
   elasticsearch:
-    image: elasticsearch:7.9.0
+    image: elasticsearch:7.15.1
```
Comment (edevosc2c) [inline docker-compose.yml]: `7.15.x` versions seems to only be available under the docker image: `docker.elastic.co/elasticsearch/elasticsearch`.

So: `docker.elastic.co/elasticsearch/elasticsearch:7.15.1`

Or we have to use `7.16.x` versions which are available: https://hub.docker.com/_/elasticsearch/tags?page=1&name=7.16

**Review by edevosc2c — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (pmauduit) [reply docker-compose.yml]: right, I'll adapt the registry to the official elastic one, thanks

Comment (pmauduit) [reply docker-compose.yml]: I'd prefer to harmonize versions between georchestra's ansible playbook, docker ... and having the same server version as the client being used in geonetwork4.

**Review by edevosc2c — APPROVED** [review]

**Pull request merged**